libtransact/src/contract/archive/mod.rs

```diff
@@ -117,7 +117,8 @@ impl SmartContractArchive {
         version: &str,
         paths: &[P],
     ) -> Result<SmartContractArchive, Error> {
-        let file_path = find_scar(name, version, paths)?;
+        let version_req = VersionReq::parse(version)?;
+        let file_path = find_scar(name, version_req.clone(), paths)?;
```

Comment:
Can `find_scar` just borrow the version req?

Reply:
Yep, updated